apps/hubble-stats/utils/date.ts

```diff
@@ -24,10 +24,11 @@ export const getEpochNow = () => getEpochFromDate(new Date());
 /**
  * Total number of days have passed since the starting epoch
  */
+
+const EPOCH_START = 1692144000;
+
 export const getEpochStart = () => {
-  // Currently set this as the epoch that's exactly 7 days ago
-  const epoch7d = getEpochNow() - 6 * EPOCH_DAY_INTERVAL;
-  return Math.floor(epoch7d / EPOCH_DAY_INTERVAL) * EPOCH_DAY_INTERVAL;
+  return EPOCH_START;
```

Comment:
I think we must get the date from all the history instead of recent days.
Reasons include:
- If we do not the chart with all history data, users might get concerned with the data displayed in the Key Metric Table and the Chips in the header since the data for these components are also all history data
- [Uniswap](https://info.uniswap.org/#/) also displays the full history data in their charts. For starting time, they also hardcoded it, you can check that [here](https://github.com/Uniswap/v3-info/blob/master/src/data/pools/chartData.ts#L56).

Reply:
If so, please consider parameterizing this into an environment variable and add a default value for it.